docs/OverseeingIssuePrLifeCycle.md

```diff
@@ -18,18 +18,21 @@ When an issue comes in,
 
 * A PR should have,
   * **1 or more `reviewers`** (See [Reviewing PRs](ReviewingPrs.md) for more info)
-  * **no more than one `assignee`**: The `assignee` field can be used to indicate the main reviewer when there are 
-    multiple reviewers and it is not clear who is the main reviewer. Otherwise it can be left empty.
+  * **all reviewers should be selected as `assignees`**
+  
+  > Duplicating `reviewers` as `assignees` is needed because GitHub filter syntax cannot filter by reviewers yet.
```

Comment:
Apparently, the filter works. It's this button (which I expected to be same as the filter) that doesn't work. Anyway, i'll create an issue to update this doc.

![image](https://cloud.githubusercontent.com/assets/1673303/22359698/781f47de-e484-11e6-8b23-3c019c8924ad.png)